### DISCUSSION TITLE
How to get the bit rate for my stream from API

### DISCUSSION THREAD

**Discussion by quizmastergary** [AzuraCast/AzuraCast#7467] [Q&A]
Hi everyone,

I'm running a radio website and am trying to use the Azura API to get the mount points available and their respective bit rates to put them on clickable buttons for the available streams.

We start the streams on the Azura server and then on another computer, use Sound Empire Caster to encode the streams at different bit rates for each mount point.

In the server statistics page the correct bit rates that are being served show up, but I can't find this anywhere in the API (I only seem able to get whatever the Auto DJ bit rate it set to, which could be different).

Can anyone point me in the right direction of the correct API call to use to get this info?!

**Comment by BusterNeece** [comment]
We don't currently store the bit rate that's reported back to us from Icecast/Shoutcast endpoints, as for a majority of our use-cases that information is redundant to our own internal knowledge of what the bitrate _should_ be (as generated by the AutoDJ).

Your situation is an interesting edge-case where it would make sense to document and report the mount point bitrate currently in use on the stream, even though our own AutoDJ isn't broadcasting to it directly. I'm surprised that this hasn't come up more frequently in the almost 9 years AzuraCast has been around, honestly.

Since it's not an existing feature, it's likely a good candidate for inclusion over at https://features.azuracast.com/.
